tripy/tests/integration/test_groupnorm.py

```diff
@@ -47,15 +49,15 @@ def test_groupnorm_accuracy(self, torch_dtype, tp_dtype, input_shape, num_groups
             dtype=tp_dtype,
         )
 
-        tp_groupnorm.weight = tp.Parameter(groupnorm.weight)
-        tp_groupnorm.bias = tp.Parameter(groupnorm.bias)
+        tp_groupnorm.weight = tp.Parameter(groupnorm.weight.detach())
```

Comment:
can we do this internally? See https://github.com/NVIDIA/TensorRT-Incubator/pull/72#discussion_r1710444219.

Reply:
If we do this internally, we have to import torch, which is not ideal. Also we don't want to do this for all other training frameworks, just let user clear the gradients before converting to tripy.

Reply:
What's the consequence of not doing `detach()`?

Reply:
Torch tensor's dlpack interface will raise an error:
```
File "/usr/local/lib/python3.10/dist-packages/torch/_tensor.py", line 1470, in __dlpack__
    raise RuntimeError(
RuntimeError: Can't export tensors that require gradient, use tensor.detach()
```
It is informative.